wgpu-types/src/lib.rs

```diff
@@ -7683,7 +7683,7 @@ pub enum Dx12Compiler {
         dxil_path: PathBuf,
     },
     /// The statically-linked variant of Dxc.
-    /// The `static-dxc` feature is required to use this.
+    /// The `static-dxc` feature is required to use this. Not available on windows-aarch64.
```

Comment:
style:

```suggestion
    /// The `static-dxc` feature is required to use this. Not available on `windows-aarch64`.
```

Reply:
That's fine, if we're doing quotes though it needs to be `aarch64-windows-*` or something